lib/loader.js

```diff
@@ -6,18 +6,14 @@ var loaderUtils = require("loader-utils");
 var processCss = require("./processCss");
 var getImportPrefix = require("./getImportPrefix");
 var compileExports = require("./compile-exports");
-var createResolver = require("./createResolver");
 
 
 module.exports = function(content, map) {
-	if(this.cacheable) this.cacheable();
 	var callback = this.async();
 	var query = loaderUtils.getOptions(this) || {};
-	var root = query.root;
-	var moduleMode = query.modules || query.module;
-	var camelCaseKeys = query.camelCase || query.camelcase;
+	var moduleMode = query.modules;
```

Comment:
because this is a refactor, shouldn't we use `const` and `let` instead?

Reply:
@shellscape also let's do it other PR, when we ship webpack-default in this repo

Reply:
OK